### DISCUSSION TITLE
ROCKET_TLS={certs....could not be parsed

### DISCUSSION THREAD

**Discussion by M4st3rITA** [dani-garcia/vaultwarden#2017] [Q&A]
### Subject of the issue

since yesterday, it is no longer possible to use externally created SSL certificates; the rest of the server is unaffected; I am using NGINX Proxy Manager
### Deployment environment

* vaultwarden version:
1.22.2

* Install method:

* Clients used: 

* Reverse proxy and version: 
NGINX Proxy Manager v2.9.9
* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

### Steps to reproduce

after starting up the docker, as usual, Vaultwarden is not reachable and a 502 bad gateway error is returned; the docker log reads as follows: 
[2021-10-02 07:26:17.075][rocket::config::error][ERROR] environment variable ROCKET_TLS={certs="/ssl/npm-8/fullchain.pem",key="/ssl/npm-8/privkey.pem"} could not be parsed

Logger failed to initialize: attempted to set a logger after the logging system was already initialized

**Comment by M4st3rITA** [comment]
Compared to the past few days, nothing has been changed, no settings, no variables.

**Comment by BlackDex** [comment]
Is this cert a Lets Encrypt cert?
If so, it could have something to do with this: https://letsencrypt.org/2021/10/01/cert-chaining-help.html

**Comment by M4st3rITA** [comment]
Yes, definitely...I'm using Let's Encrypt certs.
How can I resolve this...?
I already tried to use the HTTP version of Vaultwarden without success because of the security policy of the browser

**Comment by BlackDex** [comment]
Well if you use a reverse proxy, you could just have the reverse proxy offload the ssl. That might work.

Else i think you are using the Alpine image, you might want to switch to the Debian image.
